### PR TITLE
Adding travis.yml

### PR DESCRIPTION
**Pull request by scandindian**
Adding this for tests. For this issue https://github.com/fossasia/connfa-android/issues/13

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    # Support library
+    - extra-android-support
+    # Latest artifacts in local repository
+    - extra-google-m2repository
+    - extra-android-m2repository
+    - android-sdk-license-.+
+    - '.+'
+    # Specify at least one system image
+    - sys-img-armeabi-v7a-google_apis-$ANDROID_API_LEVEL
+    - sys-img-armeabi-v7a-android-$ANDROID_API_LEVEL_23
```
Comment (iamareebjamal) [inline .travis.yml]: If there are no UI tests, there is no need to use images. See orga app travis yml

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+before_script:
+  - chmod +x gradlew
```
Comment (iamareebjamal) [inline .travis.yml]: Add `./gradlew build` or `./gradlew check` in `script` section

**Review by iamareebjamal — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 2 above)
Comment (scandindian) [reply .travis.yml]: @iamareebjamal Should I replace before_sript section with that or add separately?

Comment (iamareebjamal) [reply .travis.yml]: Keep before script section, and add additional script section

Comment (scandindian) [reply .travis.yml]: @iamareebjamal Okay, I have added that. Can you check again, please?

**Comment by scandindian** [comment]
@iamareebjamal Can you check again, please?

**Comment by iamareebjamal** [comment]
@heysadboy The build is failing ` You have not accepted the license agreements of the following SDK components`

Other than that, it is fine

File: app/build.gradle [hunk 3]
```diff
@@ -56,6 +56,10 @@ android {
         preDexLibraries = false
         javaMaxHeapSize "4g"
     }
+
+    lintOptions {
+        abortOnError false
```
Comment (iamareebjamal) [inline app/build.gradle:53]: Instead of suppressing error, please try to solve it. Suppressing errors in build can result in crashes on runtime

**Review by iamareebjamal — CHANGES_REQUESTED** [review]

File: app/build.gradle (hunk 3 above)
Comment (scandindian) [reply app/build.gradle:53]: @iamareebjamal I tried removing without this and it fails to build. The error was due to the SDK version for parallax SDK and automatic signing, I have changed the SDK version to match that of app.

Comment (iamareebjamal) [reply app/build.gradle:53]: OK

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**